Solutions/Corvus.HighPerformance/Corvus.HighPerformance/Utf8/ValueStringBuilder.cs

```diff
@@ -153,6 +153,37 @@ public ReadOnlyMemory<byte> CreateMemoryAndDispose()
 
     /// <summary>Returns the underlying storage of the builder.</summary>
     public Span<byte> RawBytes => _bytes;
+    
+    /// <summary>
+    /// Slice the builder to remove the first <paramref name="start"/> bytes.
+    /// </summary>
+    /// <param name="start">The number of bytes to slice from the start.</param>
+    public void Slice(int start)
```

Comment:
Initially I was a bit confused by the test drivers that called this because I was not expecting `Slice` to be side-effectful. When I think of `Slice` on a span or memory, I expect it to return me a new view without changing the underlying thing.

I'm wondering if we should think about giving this a name that makes it clear that this modifies the target.